static/js/email_cert.js

```diff
@@ -0,0 +1,19 @@
+function log(obj) {
+    response = JSON.stringify(obj);
+    if (response.search("sent")){
```

Comment:
Response `status` from Mandrill API can be `"sent"` while it still returns a value in `"reject_reason"`, like `"hard-bounce"` which is another failure state as far as the user is concerned.  This probably shouldn't happen for ExtraCare but  might be good to check that there is no "reject_reason", as well.  An example from https://mandrillapp.com/api/docs/messages.html

```
[
    {
        "email": "recipient.email@example.com",
        "status": "sent",
        "reject_reason": "hard-bounce",
        "_id": "abc123abc123abc123abc123abc123"
    }
]
```

Reply:
@bryanlandia great point wasn't aware of it i write a condition for it